ocaml/utils/language_extension.mli

```diff
@@ -103,11 +103,27 @@ val with_disabled : 'a t -> (unit -> unit) -> unit
     way. *)
 val restrict_to_erasable_extensions : unit -> unit
 
-(** Permanently ban all extensions; used for [-disable-all-extensions] to
-    ensure that some code is 100% extension-free.  When called, disables any
+(** Permanently ban all extensions; used for [-disable-all-extensions] to ensure
+    that some code is 100% extension-free.  When called, disables any
     currently-enabled extensions, including the defaults.  Causes any future
-    uses of [set ~enabled:true], [enable], and their [with_] variants to
-    raise; also causes any future uses of [only_erasable_extensions] to raise.
+    uses of [set ~enabled:true], [enable], and their [with_] variants to raise;
+    also causes any future uses of [restrict_to_erasable_extensions] to raise.
     The [is_enabled] function will still work, it will just always return
     [false].*)
 val disallow_extensions : unit -> unit
+
+(**/**)
+
+(** Special functionality that can only be used in "pprintast.ml" *)
+module For_pprintast : sig
+  (** A fully-polymorphic function that can wrap printers from "pprintast.ml" *)
+  type printer_exporter =
+    { export_printer :
```

Comment:
I might go verbose and name it `print_with_extensions_maximally_enabled`. "Exporting" to me wants to suggest adding to some global registry, even though I think I see what you're going for.